Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -48,10 +48,17 @@ class AnnotationTrackingState {
 public:
     void startAnnotationTracking(RetainPtr<PDFAnnotation>&&, const WebEventType&, const WebMouseEventButton&);
     void finishAnnotationTracking(const WebEventType&, const WebMouseEventButton&);
-    const PDFAnnotation* trackedAnnotation() const { return m_trackedAnnotation.get(); }
+    const RetainPtr<PDFAnnotation>& trackedAnnotation() const { return m_trackedAnnotation; }
+    bool isBeingHovered() const
```

Comment:
would rather not have multiline implementations in the giant header, if possible.

Reply:
Changed